pkgs/development/python-modules/pyzbar/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, buildPythonPackage, fetchFromGitHub, numpy, pillow, zbar, pytestCheckHook }:
+
+buildPythonPackage rec {
+  pname = "pyzbar";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "NaturalHistoryMuseum";
+    repo = "pyzbar";
+    rev = "v${version}";
+    sha256 = "8IZQY6qB4r1SUPItDlTDnVQuPs0I38K3yJ6LiPJuwbU=";
+  };
+
+  propagatedBuildInputs = [ zbar pillow numpy ];
+
+  checkInputs = [ pytestCheckHook ];
+
+  # find_library doesn't return an absolute path
+  # https://github.com/NixOS/nixpkgs/issues/7307
+  postPatch = ''
+    substituteInPlace pyzbar/zbar_library.py --replace "find_library('zbar')" "'${zbar.lib}/lib/libzbar.so.0'"
```

```suggestion
    substituteInPlace pyzbar/zbar_library.py \
      --replace "find_library('zbar')" "'${lib.getLib zbar}/lib/libzbar.so.0'"
```
This will also not work on darwin.